misk-hibernate-testing/src/main/kotlin/misk/jdbc/StartVitessService.kt

```diff
@@ -46,9 +51,45 @@ class VitessCluster(val config: DataSourceConfig, moshi: Moshi = Moshi.Builder()
         { it.fileName.toString() },
         { keyspaceAdapter.fromJson(it.resolve("vschema.json").source().buffer())!! })
   }
+
+  /**
+   * Connect to vtgate.
+   */
+  fun connect() = dataSource().connection
```

Comment:
is there a lifecycle on this that callers need to consider? The method signature looks like it should need to close something, but dataSource() seems like a method that doesn’t. (Maybe it should be openDatasource() ?)

Reply:
I like the word “open” on resources that need to be closed. Maybe `openConnection` ?

Reply:
`dataSource()` doesn't have a lifecycle it's basically a "connection factory". `openConnection` sgtm.